From 3780509220fc7f37ff70e41429fc11af1af00f61 Mon Sep 17 00:00:00 2001
From: Peter Yin <peteryin.openbmc@gmail.com>
Date: Fri, 14 Jun 2024 13:30:26 +0800
Subject: [PATCH 1003/1010] ARM: dts: aspeed: Harma: add lpc_pcc device

Add lpc_pcc node for AMD post code

There is no lpc_pcc driver in upstream,
so it only patch in Meta tree.

Signed-off-by: Peter Yin <peteryin.openbmc@gmail.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
index ec7a4bde1333..279ed49eee9e 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
@@ -170,6 +170,16 @@ flash@0 {
 	};
 };
 
+&lpc_pcc {
+	rec-mode = <0x1>;
+	port-addr = <0x80>;
+	port-addr-hbits-select = <0x1>;
+	port-addr-xbits = <0x3>;
+	dma-mode;
+	A2600-15;
+	status = "okay";
+};
+
 &kcs2 {
 	status = "okay";
 	aspeed,lpc-io-reg = <0xca8>;
-- 
2.44.2

